lidar_point_pillars/include/lidar_point_pillars/point_pillars.h

```diff
@@ -33,9 +33,18 @@
 #include <algorithm>
 #include <limits>
 
-// headers in TensorRT
-#include "NvInfer.h"
-#include "NvOnnxParser.h"
+#ifdef TVM_IMPLEMENTATION
+  // headers in TVM
+  #include <tvm_utility/pipeline.h>
+  #include "point_pillars_pfe/pfe_tvm_pipeline.h"
+  #include "point_pillars_rpn/rpn_tvm_pipeline.h"
```

Comment:
If you instead place these files in `include/lidar_point_pillars/tvm_pfe` and `include/lidar_point_pillars/tvm_rpn` then you can include them as `lidar_point_pillars/tvm_pfe/pfe_tvm_pipeline.h` and `lidar_point_pillars/tvm_rpn/rpn_tvm_pipeline.h` without using `ros::package`.

Reply:
I'm not sure I get it. If we are talking about the `ros::package` in `point_pillars_ros.cpp` then my understanding is that they target `tvm_models/` files, not those headers.

Reply:
Sorry, I think I was very tired when I reviewd this the first time. However, I think I assumed that this file also used `ros::package` since two lines below this, `<ros/package.h>` is included. Please remove that include if it isn't used.

Reply:
Removed